src/Arcus.WebApi.Logging.AzureFunctions/Correlation/AzureFunctionsInProcessHttpCorrelation.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Diagnostics;
+using Arcus.Observability.Correlation;
+using Arcus.WebApi.Logging.Core.Correlation;
+using Arcus.WebApi.Logging.Correlation;
+using GuardNet;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Arcus.WebApi.Logging.AzureFunctions.Correlation
+{
+    /// <summary>
+    /// Represents an <see cref="HttpCorrelationTemplate{THttpRequest,THttpResponse}"/> implementation
+    /// that extracts and sets HTTP correlation throughout Azure Functions (in-process) HTTP trigger applications.
```

Comment:
Why is there a difference in in-process & out-of-process correlation ?

Reply:
Because of the type and hosting functionality it uses.
In-process uses the same `HttpRequest`/`HttpResponse` types, but does not have middleware like a regular ASP.NET Core application.
Isolated uses `HttpRequestData`/`HttpResponseData` types, but has middleware like a regular ASP.NET Core application.